parsec/core/types/manifest.py

```diff
@@ -195,6 +196,10 @@ class LocalManifestType(Enum):
     LOCAL_USER_MANIFEST = "local_user_manifest"
 
 
+T = TypeVar("T", bound="LocalManifest")
```

Comment:
`LocalManifestType` ?

Reply:
Unfortunately  this name is already used:
```python
class LocalManifestType(Enum):
    LOCAL_FILE_MANIFEST = "local_file_manifest"
    LOCAL_FOLDER_MANIFEST = "local_folder_manifest"
    LOCAL_WORKSPACE_MANIFEST = "local_workspace_manifest"
    LOCAL_USER_MANIFEST = "local_user_manifest"
```

Maybe i can name it `LocalManifestTypeVar`